test/test_helper.rb

```diff
@@ -32,7 +32,11 @@ class TestCase
     include RubyLsp::TestHelper
 
     def dummy_root
-      File.expand_path("#{__dir__}/dummy")
+      result = File.expand_path("#{__dir__}/dummy")
+      unless result[0] == "/"
```

Comment:
The path returned on windows doesn't have the leading `/`.

Reply:
But why? The correct path on Windows wouldn't have that leading forward slash. Is it because of some URI being incorrect? If that's the case, then the solution is to use the Ruby LSP's URI builder `URI::Generic.from_path`, which already handles Windows URIs.

Reply:
Spoke to soon, still getting some windows failures.